sympy/tensor/array/ndim_array.py

```diff
@@ -586,6 +587,10 @@ def _check_index_for_getitem(self, index):
 
         return index
 
+    def _eval_evalf(self, **options):
+        """Apply evalf() to each element of self."""
+        return self.applyfunc(lambda i: i.evalf(n, **options))
```

Comment:
I guess both `*args` and `**kwargs` need to be used to make this work.

Reply:
Oops, that's my mistake for not replacing all the original options (I missed the n). I have updated it now to make use of the args and kwargs parameters.